### ISSUE TITLE
Structure block GUI's wont open

### ISSUE THREAD

**Issue by michael33997788** [ldtteam/minecolonies#4398]
### Minecolonies version
Minecolonies-0.10.732-BETA-universal

Version: 
1.15.2

### Expected behavior
open up GUI
Right clicking on a structures block (example: Farm structure block) would open up the GUI

### Actual behavior
will not open GUI

Right clicking the structures block does not open up the GUI for it

### Steps to reproduce the problem

(not sure if its 1.15.2 only or not)

1. Download the mod.
2. if you force Minecraft to shutdown (example: Clicking X, using Task Manager, etc..) and rejoin any world the blocks no longer bring up a GUI when right clicked.
3. Cant use the GUI again unless i Reinstall the mod.

### Logs: Latest.log and crashlog at https://gist.github.com/
idk how to get logs

**Comment by michael33997788** [comment]
Ive had this exact problem before but it was not fixed properly.
(Commenting because i forgot to add this in)

**Comment by Raycoms** [comment]
Does /mc colony list display any colony?

**Comment by michael33997788** [comment]
It displays my colony name when i type that
the colony still functions properly and all the people continue there jobs but the GUI's will not work

**Comment by michael33997788** [comment]
my hand moves as if its attempting to interact but it still does nothing

**Comment by Raycoms** [comment]
That is very weird. Do you have a 100% reproduce for this. Or is it randomly after closing it?

**Comment by michael33997788** [comment]
this has happened to me at least like 10 times and in the same way its always after shutting down minecraft without the exit game option in the main menu and opening it back up again

**Comment by michael33997788** [comment]
it happens in alot of recent versions of the mod for 1.15.2

**Comment by Raycoms** [comment]
I'll try to break it then =D

**Comment by MineShadow7** [comment]
Hello! I have the same issue here. I think I  can reproduce a temporary fix for this. Maybe this has to do with forge or something, but the mod just doesn't load properly. You have to restart Minecraft. Maybe even a couple of times.

**Comment by michael33997788** [comment]
ive not tried restarting minecraft more than 2 times after it breaks but when i reinstall the mod it starts working properly again (Btw thanks for the help so far)

**Comment by Raycoms** [comment]
I can't reproduce this.

**Comment by michael33997788** [comment]
thats odd

**Comment by michael33997788** [comment]
i'm using the latest version of forge 1.15.2 i'm also using the latest version of minecolonies 1.15.2. all i did was force shutdown minecraft by clicking the X in the top right corner and when i got back in it was broke it will not let me interact with any of the structure blocks.

**Comment by Raycoms** [comment]
any possibility you can send me that world?

**Comment by michael33997788** [comment]
uhh sure ill try
 
not sure if it would be bad or not but i have a few other mods installed. but pretty basic ones

**Comment by michael33997788** [comment]
[colony - Copy.zip](https://github.com/ldtteam/minecolonies/files/4303920/colony.-.Copy.zip)

**Comment by michael33997788** [comment]
the mods i have are Obfuscate , mr.crayfish furniture, vehicle, and better wolves mods and lastly the minecolonies / structurize mods

**Comment by Raycoms** [comment]
Kk, I should be able to check on that tomorrow. Going to bed now.

**Comment by michael33997788** [comment]
alright good night thank for the help

**Comment by Raycoms** [comment]
Loading your world worked fine for me. Could intract with everything.

**Comment by jolandepoel** [comment]
The steps to reproduce included closing Minecraft via the x of the window instead of quit/disconnect, and then the next time you played it would not show a gui.

**Comment by empyers** [comment]
Also had this happen. I also noticed all my colonists seemed to disappear (Except for the tavern owner). It happened after a crash when I jumped into the Twilight Forest portal (using the latest Eternal modpack 1.3.4). When I re-opened the game the next day this similar bug happened.

I tried closing Minecraft and reopening it; completely closing the program and relaunching the modpack from Twitch, worked! Colonists showed up again and the colony blocks worked.
